Makefile

```diff
@@ -1,3 +1,16 @@
+ROOT=$(shell pwd)
+
+test: test-unit test-integration
+
+test-unit:
+	@echo "\nRunning unit tests..."
+	@NODE_ENV=test mocha test/integration test/structure test/support test/unit --recursive
+
+test-integration:
+	@echo "\nRunning integration tests..."
+	rm -rf node_modules/waterline-adapter-tests/node_modules/waterline;
+	ln -s $(ROOT) node_modules/waterline-adapter-tests/node_modules/waterline;
```

Comment:
Key change to ensure we are using the latest version of waterline core (which is the target of these tests!).